code/modules/uplink/uplink_items.dm

```diff
@@ -1417,6 +1417,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
 	restricted = TRUE
 
+
+/datum/uplink_item/device_tools/suspiciousphone
+	name = "Protocol CRAB-17 Phone"
+	desc = "The Protocol CRAB-17 Phone, a phone borrowed from an unknown third party, it can be used to crash the space market, funneling the losses of the crew to your bank account.\
+	The crew can move their funds to a new banking site though, unless they HODL, in which case they deserve it."
+	item = /obj/item/suspiciousphone
+	cost = 7
```

Comment:
Alternatively, since the only purpose of this item is to make a joke which is by now more than a year old, we could go the Syndicate Ballon route: 
```suggestion
	cost = 20
```

Reply:
why are you still posting you salty literaly-who